.eslintrc.js

```diff
@@ -1,27 +1,26 @@
 module.exports = {
-  "extends": [
-    "airbnb",
-    "plugin:prettier/recommended",
-    "plugin:jsx-a11y/recommended"
+  extends: [
+    'airbnb',
+    'plugin:prettier/recommended',
+    'plugin:jsx-a11y/recommended',
   ],
-  "env": {
-    "es6": true,
-    "browser": true,
-    "jest": true,
-    "node": true
+  env: {
+    es6: true,
+    browser: true,
+    jest: true,
+    node: true,
   },
-  "parser": "babel-eslint",
-  "parserOptions": {
-    "ecmaVersion": 2017,
-    "sourceType": "module",
-    "ecmaFeatures": {
-      "jsx": true
-    }
+  parser: 'babel-eslint',
+  parserOptions: {
+    ecmaVersion: 2017,
+    sourceType: 'module',
+    ecmaFeatures: {
+      jsx: true,
+    },
   },
-  "plugins": ["prettier", "json", "jsx-a11y"],
-  "rules": {
-    "react/jsx-one-expression-per-line": "off",
-    "jsx-a11y/label-has-associated-control": "off",
```

Comment:
For clarity outside of the linting - the actual code change is removing `"jsx-a11y/label-has-associated-control": "off",`